strapi/strapi
# [chore][redux]: add redux dev-tools to launch in dev mode

### What does it do?

* adds a check to connect `redux-dev-tools` to our redux store if `NODE_ENV === 'development'`

### Why is it needed?

* makes understanding the flows of the redux store a lot easier in development

### How to test it?

* install the chrome extension
* run the example
* check that dev-tools has found `StrapiApp`
